tern/analyze/default/container/single_layer.py

```diff
@@ -144,6 +144,10 @@ def analyze_first_layer(image_obj, master_list, redo):
             # unmount
             rootfs.undo_mount()
             rootfs.unmount_rootfs()
+        # distroless images do not have a shell or binaries in the base layer
+        # we see what the OS is reported as instead
```

Comment:
If we can find `/var/lib/dpkg/status.d` on the filesystem then we can use that as an indication of a distroless base image.